arches/app/templates/login.htm

```diff
@@ -26,93 +9,97 @@
 
 <div id="container" class="cls-container arches-login">
 
-	<!-- BACKGROUND IMAGE -->
-	<!--===================================================
-	<div id="bg-overlay" class="bg-img img-login"></div>
-	-->
+    <!-- BACKGROUND IMAGE -->
+    <!--===================================================
+    <div id="bg-overlay" class="bg-img img-login"></div>
+    -->
```

Comment:
The login page also has changed. In the accessibility branch, the placeholder is now outside of the input and is quite small.

<img width="356" alt="image" src="https://github.com/archesproject/arches/assets/1877663/be0ca561-f63b-424b-8475-fc908c62b4d4">

<img width="353" alt="image" src="https://github.com/archesproject/arches/assets/1877663/94bbbe58-21ee-4991-965d-bec270aeeff5">

Reply:
Let me know what you think of these changes